### ISSUE TITLE
[Feature] Configuration "get just main pools without childs"

### ISSUE THREAD

**Issue by agospher4eg** [tomaae/homeassistant-truenas#13]
Thanks for great work! I have more then 50 entities from truenas datasets. Is it possible to get just main datasets (pools)?

**Comment by tomaae** [comment]
No, since they are all datasets. Many of them may be useful for people.
You can always hide or disable ones you dont need if they are in the way.
Pools are different entities, you can find them in system category.

**Comment by agospher4eg** [comment]
pools in the system menu doesn`t have Used and Available for my percentage "battery entity". How to disable 49 datasets and leave just one?

**Comment by tomaae** [comment]
You can find those options in entity settings:
![image](https://user-images.githubusercontent.com/23486452/163187821-9ba244fa-7b12-4aa9-9353-6d72e0a6673b.png)

**Comment by agospher4eg** [comment]
Thank you! How often he scans for new datasets? Or he did for the first time and won`t scan any more?

**Comment by tomaae** [comment]
there is no dataset scan. everything gets updated on every pull.
but once you disabled or hidden the entity, it will stay that way. HA will discard all data when disabled. Hidden will still update, you just wont see it in dashboard.

**Comment by agospher4eg** [comment]
ok
